src/BcPrebidVast.js

```diff
@@ -914,11 +917,19 @@ var prebidVastPlugin = function (player) {
 				if (_adRenderer === 'ima') {
```

Comment:
Shouldn't you be comparing to the constants in this if/else stack, instead of the strings themselves (`_rendererNames.IMA`, not `'ima'` -- `_rendererNames.CUSTOM`, not `'custom'`)

Reply:
Created Contants.js file and use renderer names from this file.